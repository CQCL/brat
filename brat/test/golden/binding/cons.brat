-- On the left hand side of a function definitino, a cons cell's
-- head and tail must be bound to separate variables
uncons :: { List(Int) -> Int, List(Int) }
uncons(cons(x, xs)) = x, xs

badUncons(List(Int)) -> Int, List(Int)
badUncons(cons(stuff)) = stuff
