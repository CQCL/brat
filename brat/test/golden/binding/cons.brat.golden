Error in badUncons@FC {start = Pos {line = 7, col = 10}, end = Pos {line = 7, col = 31}}:
  Internal error: Expected empty unders after abstract, got:
  (tail :: List(Int))

